src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt

```diff
@@ -462,9 +464,9 @@ fun Strings.ja(): String? = when (this) {
     FilterStarred -> "スターマークのあるエントリのみ表示"
     FilterUnstarred -> "スターマークのないエントリのみ表示"
     FilterStarIgnored -> "スターマークを参照しない"
-    FilterDone -> "完成のエントリのみ表示"
+    FilterDone -> "完成したエントリのみ表示"
     FilterUndone -> "未完成のエントリのみ表示"
-    FilterDoneIgnored -> "完成状態を参照しない"
+    FilterDoneIgnored -> "完成状態でフィルターしない"
```

Comment:
`FilterStarIgnored` and `FilterDoneIgnored` should use similar phrases for consistency